pkg/ddl/backfilling_dispatcher.go

```diff
@@ -328,6 +353,41 @@ func generateNonPartitionPlan(
 	return subTaskMetas, nil
 }
 
+func (dsp *BackfillingDispatcherExt) getWriterMemSize(backfillMeta *BackfillGlobalMeta) (uint64, error) {
+	failpoint.Inject("mockWriterMemSize", func() {
+		failpoint.Return(1*size.GB, nil)
+	})
+	jobMeta := &backfillMeta.Job
+	_, tbl, err := dsp.d.getTableByTxn((*asAutoIDRequirement)(dsp.d.ddlCtx), jobMeta.SchemaID, jobMeta.TableID)
+	if err != nil {
+		return 0, err
+	}
+	_, writerCnt := expectedIngestWorkerCnt()
+	indexInfos := make([]*model.IndexInfo, 0, len(backfillMeta.EleIDs))
```

Comment:
The usage of indexInfos just is len(), if so, an int `cnt` is enough?

Reply:
nice catch, will fix it